pitch.md

```diff
@@ -23,38 +89,146 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+The user should be able to describe elements of a specific graph that they want.
+They will be able to generate very commonly used elements of economics graphs, such
+as supply and demand curves, equilibrium point, production possibilities curves,
+and so on, with a single function call. Then, they would be able to customize
+each of these built-in elements if needed, with things such as slope and
+intercepts being passed in as optional parameters. 
+
+In addition, there would be other properties they can generate such as axes
+values, titles, axes intersection points, and so on.  I could imagine further, 
+more specific customizations being possible to give the DSL an almost secondary
+purpose as a graphing language; however, we'd want to focus and make it as easy
+as possible for users to create graphs for economics specifically first.
```

Comment:
Have you thought about making the language primitives sufficient for a more general graphing language, and then building the economics stuff on top of that, like a standard library? That might make the language more extensible, since users interested in, say, calculus, could go in and build their own calculus library in the same way that the econ library was built. On the other hand, a general graphing language might be a bit too broad-scoped for our purposes in this class.